js/containers/portal-dashboard/answer.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { getAnswersByQuestion } from "../../selectors/report-tree";
 import { connect } from "react-redux";
-import { getAnswerType, getAnswerIconId, AnswerProps } from "../../util/answer-utils";
+import { getAnswerType, AnswerProps } from "../../util/answer-utils";
```

Comment:
`getAnswerType` is no longer used and throws a lint warning.  We can remove it from the imports.